x11-misc/light-locker/metadata.xml

```diff
@@ -10,6 +10,7 @@
     <flag name="dpms">Turn on the display on screensaver deactivation.</flag>
     <flag name="gtk3">Build with GTK+ 3 instead of GTK+ 2.</flag>
     <flag name="screensaver">Lock the screen on screensaver activation.</flag>
+    <flag name="elogind">Enable elogind support.</flag>
```

Comment:
This should be a global definition now, you don't need to define it here. (Update your master, then merge updates into your branch if repoman complains about it.)